src/ExceptionHandler.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\BigQuery;
+
+use Keboola\StorageDriver\Shared\Driver\Exception\Command\TooManyRequestsException;
+use Throwable;
+
+class ExceptionHandler
+{
+    public const TOO_MANY_REQUESTS_CODES = [429, 409];
```

Comment:
přijde mě, že ten IAM vracá občas 400, ale zatím bych to nechal uvidíme co to bude dělat.